.github/workflows/ci.yml

```diff
@@ -22,8 +22,9 @@ jobs:
       - uses: dorny/paths-filter@v3
         id: filter
         with:
+          # Should be kept in sync with the filter in the PR Reporter workflow
           filters: |
-            jsChanged: '**/src/**.js'
+            jsChanged: '**/src/**/*.js'
```

Comment:
As we found in https://github.com/preactjs/preact/pull/4523/commits/ad1fdf50c8ce83b53dcd9a9a33a24fd7abcc654b